providers/squadcast/squadcast_provider.go

```diff
@@ -0,0 +1,179 @@
+package squadcast
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/zclconf/go-cty/cty"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+)
+
+type SquadcastProvider struct {
+	terraformutils.Provider
+	accesstoken  string
+	refreshtoken string
+	region       string
+	teamName     string
+}
+
+type AccessToken struct {
+	Type         string `json:"type"`
+	AccessToken  string `json:"access_token"`
+	IssuedAt     int64  `json:"issued_at"`
+	ExpiresAt    int64  `json:"expires_at"`
+	RefreshToken string `json:"refresh_token"`
+}
+
+// Meta holds the status of the request information
+type Meta struct {
+	Meta AppError `json:"meta,omitempty"`
+}
+
+type AppError struct {
+	Status  int    `json:"status"`
+	Message string `json:"error_message,omitempty"`
+}
+
+const (
+	UserAgent = "terraform-provider-squadcast"
```

Comment:
let's change this to `terraformer-squadcast`

Reply:
Updated